Extension/src/LanguageServer/client.ts

```diff
@@ -3504,12 +3507,32 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: false
         };
 
+        if (copy) {
+            params.copyToClipboard = true;
+        }
+
         const result: CreateDeclarationOrDefinitionResult = await this.languageClient.sendRequest(CreateDeclarationOrDefinitionRequest, params);
+
+        if (result.edit === undefined) {
+            return;
+        }
         // TODO: return specific errors info in result.
-        if (result.changes === undefined) {
+        if (result.edit.changes.length === undefined && result.clipboardText) {
+
+            if (!params.copyToClipboard) {
+                util.promptCDDFailed();
+            } else {
+                try {
+                    await vscode.env.clipboard.writeText(result.clipboardText);
```

Comment:
Does this notify the user that the text was copied to clipboard to indicate that something happened?

Reply:
No, we do not need to notify the user that it's copied.  VS doesn't and it's just one more thing in the way for the user.  They ran the command and since it should be fast they should be able to trust that it just worked.  We can notify them if it fails though.